collections/_article/dev-snapshot-godot-4-2-beta-1.md

```diff
@@ -0,0 +1,382 @@
+
+But before we go into more specific changes in various editor areas, an honorable mention goes to [MewPurPur](https://github.com/MewPurPur) who has been tirelessly working on adding missing icons and optimizing existing ones ([GH-78858](https://github.com/godotengine/godot/pull/78858), [GH-78903](https://github.com/godotengine/godot/pull/78903), [GH-79431](https://github.com/godotengine/godot/pull/79431), [GH-80102](https://github.com/godotengine/godot/pull/80102), [GH-80103](https://github.com/godotengine/godot/pull/80103), [GH-80113](https://github.com/godotengine/godot/pull/80113), [GH-80129](https://github.com/godotengine/godot/pull/80129), and many more).
+
+#### Code editor
+
+We start with the code editor, which now supports code regions (for GDScript). Code regions allow you to break up scripts into named blocks, foldable and easy to navigate, without it affecting the flow of the program ([GH-74843](https://github.com/godotengine/godot/pull/74843) by [Jean-Michel Bernard](https://github.com/jmb462)). The logic behind commenting out parts of your code also becomes more predictable with improvements to the toggle comments behavior by [Michał Iwańczuk](https://github.com/iwek7) ([GH-44557](https://github.com/godotengine/godot/pull/44557)).
+
+#### 3D viewport and tools
+
+In Godot 4.0 [Ryan Roden-Corrent](https://github.com/rcorre) added convenient hotkeys to the 3D viewport which any Blender user would find familiar. Blender-style transforms, using keys like `G`, `R`, or `S`, are now improved with support for numeric inputs for each operation, and mouse wrapping ([GH-58389](https://github.com/godotengine/godot/pull/58389), [GH-59467](https://github.com/godotengine/godot/pull/59467)).
+
+If you rely more on on-screen gizmos, you will love this next feature developed by [Tomasz Chabora](https://github.com/KoBeWi). As you probably know, box collision shapes are defined as a center point and symmetrical extents, which has long limited how they can be edited. But no longer, as each side can now be extended individually within the editor viewport ([GH-71092](https://github.com/godotengine/godot/pull/71092)).
+
+While in the 3D viewport, you may notice that it contains less clutter as various auxiliary visual information is only displayed for selected objects ([GH-75303](https://github.com/godotengine/godot/pull/75303)). At the same time elements like decals and fog volumes now have extra indicators which make them easier to select via the viewport ([GH-81554](https://github.com/godotengine/godot/pull/81554)). Both of these changes were submitted by [Hugo Locurcio](https://github.com/Calinou).
+
+![The results of decluttering of the 3D viewport](/storage/blog/godot-4-2-beta/3d-viewport-decluttering.webp)
```

Comment:
```suggestion
![The results of decluttering the 3D viewport](/storage/blog/godot-4-2-beta/3d-viewport-decluttering.webp)
```

Reply:
Or "The results of the decluttering of the 3D viewport" but that's a lot of the of the's.